extensions/amp-subscriptions/0.1/viewer-subscription-platform.js

```diff
@@ -77,16 +77,26 @@ export class ViewerSubscriptionPlatform {
   /** @override */
   getEntitlements() {
     devAssert(this.currentProductId_, 'Current product is not set');
-
+    /** @type {Object} */
+    let messageData = dict({
```

Comment:
Why not just define it as `const` and only update the `encryptedDocumentKey` later?